app/controllers/spree/admin/orders_controller.rb

```diff
@@ -12,14 +12,6 @@ class OrdersController < Spree::Admin::BaseController
 
       # Ensure that the distributor is set for an order when
       before_action :ensure_distribution, only: :new
-
-      # After updating an order, the fees should be updated as well
-      # Currently, adding or deleting line items does not trigger updating the
-      # fees! This is a quick fix for that.
-      # TODO: update fees when adding/removing line items
-      # instead of the update_distribution_charge method.
-      after_action :update_distribution_charge, only: :update
```

Comment:
Does the order in which we call this make a difference? Before the change we're doing `@order.update!` first, and then recreating all the fees.

Reply:
Yeah, `order.update!` calls the `Order::Updater` class, which makes sure the totals are updated correctly after something changes. If we're changing the order's fees, we should definitely run that afterwards. And if we're rendering the result of those changes it should be done before the render...